docroot/modules/custom/mass_fields/mass_fields.module

```diff
@@ -892,12 +893,23 @@ function mass_fields_field_widget_single_element_image_image_form_alter(&$elemen
   if (in_array($element['#field_name'], ['field_featured_item_highlight', 'field_featured_item_image'])) {
     $element['#process'][] = 'mass_fields_image_widget_alt_help_text_process';
   }
+  elseif (in_array($element['#field_name'], ['field_event_image', 'field_event_logo'])) {
+    $element['#process'][] = 'mass_fields_event_image_widget_alt_help_text_process';
+  }
 }
 
 function mass_fields_image_widget_alt_help_text_process($element, &$form_state, $form) {
   if (isset($element['alt'])) {
     $text = 'If the image conveys information that is not part of the link text, describe it here. If the image is purely decorative, leave it blank. Screen readers will read the alt text first, then the link text.';
-    $element['alt']['#description'] = new \Drupal\Core\StringTranslation\TranslatableMarkup($text);
+    $element['alt']['#description'] = new TranslatableMarkup($text);
+  }
+  return $element;
+}
+
```

Comment:
@arthurbaghdas  Please add function description and purpose. 
Looks like is a helper function? If yes, please start from `_`.